src/aktualizr_repo/image_repo.h

```diff
@@ -8,6 +8,10 @@ class ImageRepo : public Repo {
   ImageRepo(boost::filesystem::path path, const std::string &expires, std::string correlation_id)
       : Repo(Uptane::RepositoryType::Image(), std::move(path), expires, std::move(correlation_id)) {}
   void addImage(const boost::filesystem::path &image_path);
```

Comment:
It feels like too much overloading of this `addImage`.

Could be helpful to give them different names.